docs/content/quickstart.md

```diff
@@ -5,52 +5,90 @@ date = "2022-05-08T14:05:02.118466Z"
-For Spin, follow [the Spin quickstart guide](https://spin.fermyon.dev/quickstart)
-or head over to GitHub and download [the latest canary release](https://github.com/fermyon/spin/releases/tag/canary).
 
-For `bart`, the Bartholomew CLI, either [follow the contribution guide](./contributing.md)
-and build it from source, or download the pre-built binary from [the latest
-GitHub release](https://github.com/fermyon/bartholomew/releases/).
+### Spin
+
+For Spin, follow [the Spin quickstart guide](https://spin.fermyon.dev/quickstart) which details how to either:
+- download the latest Spin binary release,
+- clone and install Spin using cargo, or
+- clone and build Spin from source.
 
-## Creating your first website with Bartholomew
+### Bartholomew CLI
 
-Now that you have the `bart` CLI in your path, you can use it to create your
-first website from a template:
+For the `bart` CLI, there are two options:
+- download the latest `bart` binary [release](https://github.com/fermyon/bartholomew/releases/), or
+- clone and install `bart` from source, using the following commands:
 
 ```bash
-# create a new site in the `website/` directory, based on the default Bartholomew site template
-$ bart new site --git https://github.com/fermyon/bartholomew-site-template website
-Successfully created new Bartholomew website in directory website.
-Run spin up --file website/spin.toml to start your website locally.
+$ git clone https://github.com/fermyon/bartholomew.git
+$ cd bartholomew
+$ make bartholomew
 ```
 
-This command will pull the git template locally, and you can start Spin and run
-your new website locally:
+## Templates
+
+This quickstart method uses a Bartholomew website template. So while we do require `spin` and `bart` CLI (as per the details above) everything else we need (to launch our Bartholomew CMS website) is packaged up in the Bartholomew website template. Including the `bartholomew.wasm` and the `spin_static_fs.wasm` files which take care of Bartholomew's business logic and [Spin's file server](https://github.com/fermyon/spin-fileserver) needs, respectively. We will start working with the template in the next section.
+
+## Create Your First Website
+
+We can now generate a new repository with the same directory and file structure as the template. Simply visit [the template['s location ](https://github.com/fermyon/bartholomew-site-template)on[ GitHub](https://github.com/fermyon/bartholomew-site-template). Then click on the green `Use this template` button and follow the prompts. Here are some additional details about [creating a repository from a template](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-repository-from-a-template) if you are interested.
+
+![Use template](../static/image/docs/use-template.png)
+
+## Customize Your Website
+
+You can clone and customize your own new codebase and then run the website locally, to suit your needs:
 
 ```bash
-$ cd website
+$ cd bartholomew-site-template
```

Comment:
Leaving it as `website` is probably better here, or use a placeholder like `<your-repo-name>`

Reply:
Yes, I think `<your-repo-name>` is clearest. Thanks